### ISSUE TITLE
Source event ID type inconsistent with documentation

### ISSUE THREAD

**Issue by LeoVaris** [privacysandbox/measurement-simulation#2]
The source event ID, defined [here](https://developer.android.com/design-for-safety/privacy-sandbox/attribution#register-attribution-source), should be a 64-bit **unsigned** integer.

This library [handles](https://github.com/privacysandbox/measurement-simulation/blob/main/java/com/google/rubidium/SourceProcessor.java#L93) it as a 64-bit **signed** integer. If the input contains an integer which does not fit in to the signed range, the program will fail.

**Comment by suriheemanshu** [comment]
Hello LeoVaris,

Thanks for pointing out the discrepancy. Our team is aware of this issue and it would be fixed in our next release. Please let us know if it is impacting your workflow.

**Comment by levyitay** [comment]
@suriheemanshu This was not fixed yet, correct?

**Comment by levyitay** [comment]
IMO there are 2 options to fix it:
1) Use different JSON Parser that supports uint64 (GSON maybe)?
2) not the best option imo expect source_event_id to be string and convert string to ULong something like this:
```Java
String eventIdStr = (String) jsonObject.get("source_event_id");
    
builder.setEventId(new UnsignedLong(Long.parseUnsignedLong(eventIdStr)));
```

**Comment by suriheemanshu** [comment]
This issue has been fixed in our latest tag [v0.2.1](https://github.com/privacysandbox/measurement-simulation/releases/tag/v0.2.1). We have made input data in line with the expected schema described [here](https://developer.android.com/design-for-safety/privacy-sandbox/attribution).
